jsr352/core/src/test/java/org/hibernate/search/jsr352/massindexing/JobParameterValidationListenerTest.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.jsr352.massindexing;
+
+import org.hibernate.search.exception.SearchException;
+
+import org.junit.Test;
+
+/**
+ * @author Mincong Huang
+ */
+public class JobParameterValidationListenerTest {
+
+	private JobParameterValidationListener validationListener;
+
+	@Test(expected = SearchException.class)
+	public void validateRowsPerPartition_valueIsNegative() throws Exception {
```

Comment:
I don't know if I should avoid using the underscore here? It's the only way I could imagine to make the test more readable. Please let me know if we can have a bette solution.